src/lib/operators/get_table.cpp

```diff
@@ -0,0 +1,20 @@
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "../storage/storage_manager.hpp"
+#include "get_table.hpp"
```

Comment:
Double include and strange order of includes. The corresponding header is usually included first.